### PR TITLE
Build with constructor 3.4.4

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jaimergp** [comment]
Ok @conda-forge/miniforge, this is ready for review. I've tested the GHA artifacts locally and the Windows uninstallers are patched properly, preventing the problems described in https://www.anaconda.com/blog/windows-installer-security-fix.

File: scripts/build.sh [hunk 1]
```diff
@@ -69,7 +69,7 @@ else
    EXT="sh";
 fi
 # This line will break if there is more than one installer in the folder.
-INSTALLER_PATH=$(find . -name "M*forge*.${EXT}" | head -n 1)
+INSTALLER_PATH=$(find . -iname "M*forge*.${EXT}" | head -n 1)
```
Comment (isuruf) [inline scripts/build.sh:72]: Why this change?

Comment (jaimergp) [reply scripts/build.sh:72]: I was debugging a [Windows issue with `find` not getting anything](https://github.com/conda-forge/miniforge/actions/runs/5542141642/jobs/10116439026#step:4:1933). I tried that one and it's been working ever since, but it could also be a one-off thing (maybe a race condition with the pipes?).

I'll see if removing it has any effect or not.

**Comment by isuruf** [comment]
Thanks

**Comment by jaimergp** [comment]
Thank you @isuruf, and also for the new tag! I see it's been marked as a pre-release; is that a temporary decision while we wait for something else or are we ok turning it into a proper release at some point? Right now it does the job as a repository for fixed installers than can be accessed from the (upcoming) blog post, so no rush in making a decision here, but it would be nice if new installs would start using the fixed installers more or less soonish.

**Comment by isuruf** [comment]
See the release process at https://github.com/conda-forge/miniforge#release
